### PR TITLE
Jerry development

### PR DESCRIPTION
**Pull request by fancast**
Completing HTML forms and POST requests for week 2 of portfolio.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/UserComments.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  }
+
+  public UserComments(String commentsList, long timestamp) {
+    this.commentsList = commentsList;
+    this.timestamp = timestamp;
+  }
+
+  public void addComment(String userComment) {
+    commentsList = userComment;
+  }
```
Comment (ctrokid) [inline portfolio/src/main/java/com/google/sps/data/UserComments.java]: Is this method ever getting used? I see that we have `newComments.addComment()` on line 51 of DataServlet.java, but I don't think that calls this method. Is this necessary?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -26,22 +34,58 @@
...
+    String jsonComments = convertToJsonWithGson(newComments);
     response.setContentType("application/json;");
     response.getWriter().println(jsonComments);
   }
+  
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String comment = getParameter(request, "comment-input", "");
+    allComments.addComment(comment);
```
Comment (ctrokid) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Do we still need to have the `allComments` field in this class? I'm guessing the idea is that we're storing these comments in Datastore (line 71) and shouldn't need this in-memory variable anymore 😄

File: portfolio/src/main/java/com/google/sps/data/UserComments.java (hunk 1 above)
Comment (fancast) [reply portfolio/src/main/java/com/google/sps/data/UserComments.java]: Great catch! The method is not being used. I had there when I was figuring out how to handle the comments for each user. I'll remove the class and the instances where I call it.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (fancast) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: You're right it's not needed.

**Pull request merged**